app/frontend/src/features/profile/view/About.tsx

```diff
@@ -25,27 +19,13 @@ export default function About({ user }: AboutProps) {
   return (
     <>
       <Typography variant="h1">{OVERVIEW}</Typography>
-      <LabelAndText
-        label={AGE_GENDER}
-        text={`${user.age} / ${user.gender} ${
-          user.pronouns ? `(${user.pronouns})` : ""
-        }`}
-      />
-      <LabelAndText
-        label={LANGUAGES_FLUENT}
-        text={user.languagesList.toString().replace(",", ", ")}
-      />
-      <LabelAndText label={HOMETOWN} text={user.hometown} />
-      <LabelAndText label={OCCUPATION} text={user.occupation} />
-      <LabelAndText label={EDUCATION} text={user.education} />
-      <LabelAndText
-        label={JOINED}
-        text={
-          user.joined
-            ? dateTimeFormatter.format(timestamp2Date(user.joined))
-            : ""
-        }
-      />
+      <UserAgeGenderPronouns user={user} />
+      <UserLanguages user={user} />
+      <UserHomeTown user={user} />
+      <UserOccupation user={user} />
+      <UserEducation user={user} />
+      <UserSinceJoined user={user} />
```

Comment:
I'd leave it how it was before and made the changes inline here tbh. These components aren't doing anything extra and all this does is introduce an extra layer of indirection (now whoever reads this have to open another file just to trace what's going on) without much benefit imo.

Reply:
Yep that makes sense. Would there be any benefit in having some larger components (for say LastActive/References, or Language/Gender) as these are duplicated in Search Results and About or Overview. Or just leave them all as `LabelAndText`?

Reply:
If they are styled the same as what we see in the user profile (which looks like that is the case from the screenshots), then sure, that sounds like a decent use case to me to make them separate components so they can be shared 👍

Reply:
Maybe I was unclear in communicating this, but I meant putting all of those into one component

Reply:
done, comment below :)